bsc-plugin/src/lib/rooibos/RooibosSession.ts

```diff
@@ -58,6 +46,9 @@ export class RooibosSession {
             console.log('Efficient global stubbing is enabled');
             this.namespaceLookup = this.getNamespaces(program);
             for (let testSuite of this.sessionInfo.testSuitesToRun) {
+                if (testSuite.isNodeTest) {
```

Comment:
We need to add the component file to the program before we start looking for stub and expect calls to be modified now that we use the file scopes to reliably know if a global is in scope for this suite.